tests/Strategies/HTMLTest.php

```diff
@@ -3,15 +3,33 @@
 namespace SunAsterisk\DomainVerifier\Tests\Strategies;
 
 use PHPUnit\Framework\TestCase;
+use SunAsterisk\DomainVerifier\Models\DomainVerification;
 use SunAsterisk\DomainVerifier\Strategies\HTML;
+use Mockery as m;
 
 class HTMLTest extends TestCase
 {
     public function test_it_can_verify_html_tag()
     {
         $url = 'https://domain.local';
-        $result = (new HTML)->verify($url);
+        $domain = m::mock(DomainVerification::class);
+        $html = m::mock(HTML::class);
 
-        $this->assertInternalType('bool', $result);
+        $html->shouldReceive('verify')
```

Comment:
Don't mock the `verify` method. I think we should mock partials of HTML strategy such as: `getMetaTag` method.
```suggestion
        $domainVerifiable = Mock::mock(DomainVerifiable::class);
        $domainVerifiable->shouldReceive('getKey')->andReturns($id);
        $html->shouldReceive('getMetaTag')->andReturns($metaTags);

        $result = $html->verify($website, $domainVerifiable);
        $this->assertTrue($result);
```